src/components/modals/ConfirmationModal.js

```diff
@@ -1,37 +1,64 @@
-import {button, div, h} from 'react-hyperscript-helpers';
-import {Styles} from '../../libs/theme';
+import React from 'react';
+import {div, h} from 'react-hyperscript-helpers';
 import Modal from 'react-modal';
 import CloseIconComponent from '../CloseIconComponent';
+import './ConfirmationModal.css';
+import {styled} from '@mui/material/styles';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
 
 const ConfirmationModal = (props) => {
   const {showConfirmation, closeConfirmation, title, message, header, onConfirm, styleOverride = {}} = props;
   const closeFn = () => closeConfirmation();
 
-  const confirmButton = button({className: 'cell-button hover-color', onClick: onConfirm}, ['Confirm']);
-  const computedStyle = Object.assign({}, Styles.MODAL.CONFIRMATION, styleOverride);
+  const duosBlue = 'rgb(0, 96, 159)';
+  const duosBlueHover = 'rgb(9,72,183)';
+
+  const SecondaryButton = styled(Button)(() => ({
+    fontFamily: 'Montserrat, sans-serif',
+    color: duosBlue,
+    backgroundColor: 'white',
+    borderRadius: '4px',
+    fontSize: '1.45rem',
+    borderColor: duosBlue,
+    '&:hover': {
+      borderColor: duosBlueHover,
+      color: duosBlueHover
+    },
+  }));
+
+  const PrimaryButton = styled(Button)(({theme}) => ({
+    fontFamily: 'Montserrat, sans-serif',
+    color: theme.palette.getContrastText(duosBlue),
+    backgroundColor: duosBlue,
+    borderRadius: '4px',
+    fontSize: '1.45rem',
+    '&:hover': {
```

Comment:
this really is useful, and addresses some of my concerns about using js styles ::)

Reply:
I wasn't able to use regular css with the `Button` components ... had to go this route.